docs/manual/guides/install-test-versions.de.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Testversionen installieren"
+description: "Anleitung zur Installation von Entwicklungsversionen und Release Candidates."
+url: "anleitungen/testversionen-installieren"
+weight: 20
+---
+
+
+Das Entwicklerteam von Contao hält sich an einen festgelegten [Release-Plan][releasePlan]
+bei der Entwicklung neuer Versionen (bzw. dem Support bestehender). Vor jeder geplanten
+neuen Version gibt es einen gewissen Zeitraum in dem aktiv an dieser Version entwickelt
+wird. Neue Features werden hinzugefügt, bis ein gewisser Zeitpunkt erreicht ist.
+Danach werden die ersten »Release Candidates« veröffentlicht. Wenn während der Tests
+keine weiteren Probleme gefunden werden, wird die erste stabile Version veröffentlicht.
+
+Auch wenn Contao Unit Testing einsetzt können immer unvorhergesehen Probleme und
```

Comment:
I would not write `Unit Testing` because that's not a term known to users. Also we also do have functional tests and integration tests ;) I'd just write "Auch wenn Contao über tausende an automatisierten Tests verfügt, können immer..."